charts/vector/values.yaml

```diff
@@ -88,9 +88,9 @@ podDisruptionBudget:
   # for Vector.
   enabled: false
   # podDisruptionBudget.minAvailable -- The number of Pods that must still be available after an eviction.
-  minAvailable: 1
+  # minAvailable: 1
   # podDisruptionBudget.maxUnavailable -- (int) The number of Pods that can be unavailable after an eviction.
-  maxUnavailable:
+  # maxUnavailable:
```

Comment:
Unfortunately this is a breaking change to the Helm chart, which will impact other users.

Are you able to get your setup to work with the following configuration?
```
  enabled: true
  # podDisruptionBudget.minAvailable -- The number of Pods that must still be available after an eviction.
  minAvailable:
  # podDisruptionBudget.maxUnavailable -- (int) The number of Pods that can be unavailable after an eviction.
  maxUnavailable: 1
```